services/core-api/app/api/mines/tailings/resources/tailings.py

```diff
@@ -57,48 +55,40 @@ class MineTailingsStorageFacilityResource(Resource, UserMixin):
         'eor_party_guid',
         type=str,
         help='GUID of the party that is the Engineer of Record for this TSF.',
-        location='json',
         store_missing=False)
     parser.add_argument(
         'tqp_party_guid',
         type=str,
         help='GUID of the party that is the Qualified Person for this TSF.',
-        location='json',
         store_missing=False)
     parser.add_argument(
         'notes',
         type=str,
         help='Any additional notes to be added to the tailing.',
-        trim=True,
-        location='json')
+        trim=True)
     parser.add_argument(
         'storage_location',
         type=str,
         help='Storage location of the tailings (above or below ground)',
-        location='json',
         store_missing=False)
     parser.add_argument(
         'facility_type',
         type=str,
         help='Type of facility.',
-        location='json',
         store_missing=False)
     parser.add_argument(
         'tailings_storage_facility_type',
         type=str,
         help='Type of tailings storage facility.',
-        location='json',
         store_missing=False)
     parser.add_argument(
         'mines_act_permit_no',
         type=str,
         help='Mines Act Permit Number',
-        location='json',
         store_missing=False)
     parser.add_argument(
         'is_submitting',
         type=bool,
-        location='json',
```

Comment:
I was noticing really weird stuff with tests!
I was trying to send through is_submitting=True on the test and it just was _not picking up._ Additionally it was missing a bunch of other properties, and the lat/lon would always be None (lat/lon has a lambda). It was the location=json. I don't know how it worked on the application with that still in there, but it did not with tests. Works in the application without it.